applications/nrf_desktop/configuration/nrf52840_pca10059/ble_discovery.h

```diff
@@ -0,0 +1,35 @@
+/*
```

Comment:
Other similar files have "_def" postfix. Can you add it here too?

Reply:
done